RELEASE.md

```diff
@@ -1,153 +1,116 @@
+- mark this PR as an `enhancement` if it is a minor release
+- check that everything is ok
+- merge branch
+- check that the Prepare Release PR has been correctly opened by `release-plan`
+- Merge it when you are ready to release
 
-```
-git clean -fdx
-pnpm install
-```
+### Beta release from the `beta` branch
 
-* Update the CHANGELOG.md
-  * Run `node ./dev/changelog`
-  * Copy output into `CHANGELOG.md`
-  * Edit to make clearer for consumers (remove non-consumer facing entries, etc)
-  * Ensure blueprint change diff URLs are correct
-  * Update changelog header for current release
-  * Commit
-* Release: `npx release-it`
-* Update GitHub Release with changelog contents
-* Merge `release` into `beta` branch
+- fetch latest from origin `git fetch`
+- create a new branch to merge `release` into `beta` e.g. `git checkout -B merge-release origin/beta`
+- merge release into this new branch e.g. `git merge origin/release --no-ff`
+- Update blueprint dependencies to beta
 
 ```
-git checkout -B beta --track origin/beta
-git merge origin/release
-git push origin beta
+node ./dev/update-blueprint-dependencies.js --ember-source=beta --ember-data=beta
 ```
 
-## Initial Beta Release
+- push and open a PR targeting `beta`
+- mark this PR as an `enchancement` if the next beta is a minor release
+- check that everything is ok
+- merge branch
+- check that the Prepare Release PR has been correctly opened by `release-plan`
+  - note: the release-plan config will automatically make this version a pre-release
+- Merge it when you are ready to release
 
-* Update `ember-source` and `ember-data` to latest beta
 
-```
-node ./dev/update-blueprint-dependencies.js --ember-source=beta --ember-data=beta
-```
+### Alpha release from the `master` branch
 
-* Commit
-* Send pull request to `master` branch
-* Wait for PR (for updating blueprint dependencies) to be merged
-* Checkout the `beta` branch
+- fetch latest from origin `git fetch`
+- create a new branch to merge `beta` into `master` e.g. `git checkout -B merge-beta origin/master`
+- merge release into this new branch e.g. `git merge origin/beta --no-ff`
+- Update blueprint dependencies to alpha
 
 ```
-git fetch origin
-git checkout -B beta --track origin/beta
+node ./dev/update-blueprint-dependencies.js --ember-source=alpha --ember-data=canary
 ```
 
-* Merge `master` branch into `beta`
+- push and open a PR targeting `master`
+- mark this PR as an `enchancement` if the next alpha is a minor release
+- check that everything is ok
+- merge branch
+- check that the Prepare Release PR has been correctly opened by `release-plan`
+  - note: the release-plan config will automatically make this version a pre-release
+- Merge it when you are ready to release
 
-```
-git fetch origin
-git merge origin/master
-git push origin beta
-```
 
-* Ensure you have the correct dependencies
+<!--
 
-```
-git clean -fdx
-pnpm install
-```
+TODO decide on changelog process - i.e. do we need it to be manual any more
```

Comment:
Not manual is fine so long as someone is reviewing the changelog before merging Prepare Release and we socialize that PR titles end up in the changelog. We also need to ensure we get the proper changelog and not just the PRs updating the ember-source version.